src/commands/troubleshoot/analyse/proxy-status.ts

```diff
@@ -0,0 +1,104 @@
+import { Command, Config } from "@oclif/core";
+import { execSync } from "child_process";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import TroubleShootOutputLogger from "../utils/logger.js";
+
+type DockerSettingsInterface = {
+    OverrideProxyHTTP: string;
+    OverrideProxyHTTPS: string;
+    SettingsVersion:number
+    ProxyHttpMode: string
+    ProxyHTTPMode: string
+}
+
+export default class TroubleshootProxyStatus extends Command {
+
+    static description = "Checks user's proxy configuration " +
+        "and docker proxy configuration  ";
+
+    private readonly outputLogger: TroubleShootOutputLogger;
+
+    constructor (argv: string[], config: Config) {
+        super(argv, config);
+
+        const logger: (msg: string) => void = (msg: string) => this.log(msg);
+        this.outputLogger = new TroubleShootOutputLogger(logger);
+    }
+
+    async run (): Promise<any> {
+        const result = await this.checkCHProxyConfig();
+        if (result === "success") {
+            const dockerProxyResult = await this.checkDockerSettingsConfig();
+            if (dockerProxyResult === "success") {
+                this.outputLogger.logMessage("No issues found on proxy config", "Success");
+                this.exit(0);
+            }
+        } else {
+            this.exit(1);
+        }
+    }
+
+    private checkCHProxyConfig (): Promise<string> {
+        const chProxy = process.env.CH_PROXY_HOST;
+        if (chProxy) {
+            try {
+                const output = execSync(`ping -c3 ${chProxy}`, { encoding: "utf-8" });
```

Comment:
https://github.com/companieshouse/chs-dev/blob/main/src/helpers/vpn-check.ts#L11 returns true/false whether/not on the VPN, don't need to run this separately here.